myft-digest-promo/index.js

```diff
@@ -0,0 +1,93 @@
+const myFtClient = require('next-myft-client');
```

Comment:
This file seems to be a duplicate of `my-ft-digest-promo/index.js`? (Although `myft-digest-promo` is the better folder name 👍)